bigquery_etl/cli/dag.py

```diff
@@ -0,0 +1,223 @@
+"""bigquery-etl CLI dag command."""
+
+import click
+import os
+from pathlib import Path
+import sys
+import yaml
+
+from ..query_scheduling.dag_collection import DagCollection
+from ..query_scheduling.dag import Dag
+from ..query_scheduling.generate_airflow_dags import get_dags
+from ..cli.utils import is_valid_dir, is_valid_file
+from ..metadata.parse_metadata import Metadata, METADATA_FILE
+
+dags_config_option = click.option(
+    "--dags_config",
+    "--dags-config",
+    help="Path to dags.yaml config file",
+    type=click.Path(file_okay=True),
+    default="dags.yaml",
+    callback=is_valid_file,
+)
+
+sql_dir_option = click.option(
+    "--sql-dir",
+    "--sql_dir",
+    help="Path to directory with queries",
+    type=click.Path(file_okay=False),
+    default="sql/",
+    callback=is_valid_dir,
+)
+
+output_dir_option = click.option(
+    "--output-dir",
+    "--output_dir",
+    help="Path directory with generated DAGs",
+    type=click.Path(file_okay=False),
+    default="dags/",
+    callback=is_valid_dir,
+)
+
+
+@click.group()
+def dag():
+    """Create the CLI group for the dag command."""
+    pass
+
+
+@dag.command(help="List all available DAGs",)
+@click.argument("name", required=False)
+@sql_dir_option
+@dags_config_option
+@click.option(
+    "--with_tasks",
+    "--with-tasks",
+    "-t",
+    help="Include scheduled tasks",
+    default=False,
+    is_flag=True,
+)
+def info(name, dags_config, sql_dir, with_tasks):
+    """List available DAG information."""
+    if with_tasks:
+        dag_collection = get_dags(sql_dir, dags_config)
+    else:
+        dag_collection = DagCollection.from_file(dags_config)
+
+    if name:
+        dag = dag_collection.dag_by_name(name)
+        if not dag:
+            click.echo(f"DAG {name} does not exist", err=True)
+            sys.exit(1)
+        sorted_dags = [dag]
+    else:
+        sorted_dags = sorted(dag_collection.dags, key=lambda d: d.name)
+
+    for dag in sorted_dags:
+        click.secho(click.style(dag.name, bold=True))
```

Comment:
`secho` implies `style`

```suggestion
        click.secho(dag.name, bold=True)
```